packages/e2e-tests/test/e2e.spec.ts

```diff
@@ -735,6 +735,17 @@ describe('e2e', function () {
       );
     });
 
+    it('throws the .toArray() suggestion when a user attempts to serialize a cursor', async function () {
+      await shell.executeLine('const b = db.test.find()');
+      await shell.executeLine('b.b = b');
```

Comment:
I don't quite understand this line here and what it does – the test should work just fine without it, right?

Reply:
Yeah, this is not needed here. I will remove it. Tried previously without a MongoDB document and with the plain object instead, like from the example in [NODE-6708](https://jira.mongodb.org/browse/NODE-6708).